subsys/spm/Kconfig

```diff
@@ -48,6 +48,16 @@ menuconfig IS_SPM
 	select TRUSTED_EXECUTION_SECURE
 
 if IS_SPM
+
+config SPM_DEBUG
+	bool "SPM debug mode"
+	default y if NO_OPTIMIZATIONS
+	default y if DEBUG_OPTIMIZATIONS
+	default n
+	help
+	  Disables the lock bit of the flash allowing it to be changed by a debugger
```

Comment:
Can you add that it disables the lock bit for the flash and the sram?

Reply:
added